packages/secure_storage/amplify_secure_storage_dart/lib/src/ffi/cupertino/core_foundation.bindings.g.dart

```diff
@@ -90,127 +67,18 @@ class Keychain {
-      _lookup<CFStringRef>('kSecAttrAccount');
-
-  CFStringRef get kSecAttrAccount => _kSecAttrAccount.value;
-
-  set kSecAttrAccount(CFStringRef value) => _kSecAttrAccount.value = value;
-
-  late final ffi.Pointer<CFStringRef> _kSecAttrService =
-      _lookup<CFStringRef>('kSecAttrService');
-
-  CFStringRef get kSecAttrService => _kSecAttrService.value;
-
-  set kSecAttrService(CFStringRef value) => _kSecAttrService.value = value;
-
-  late final ffi.Pointer<CFStringRef> _kSecMatchLimit =
-      _lookup<CFStringRef>('kSecMatchLimit');
-
-  CFStringRef get kSecMatchLimit => _kSecMatchLimit.value;
-
-  set kSecMatchLimit(CFStringRef value) => _kSecMatchLimit.value = value;
-
-  late final ffi.Pointer<CFStringRef> _kSecMatchLimitOne =
-      _lookup<CFStringRef>('kSecMatchLimitOne');
-
-  CFStringRef get kSecMatchLimitOne => _kSecMatchLimitOne.value;
-
-  set kSecMatchLimitOne(CFStringRef value) => _kSecMatchLimitOne.value = value;
-
-  late final ffi.Pointer<CFStringRef> _kSecReturnData =
-      _lookup<CFStringRef>('kSecReturnData');
-
-  CFStringRef get kSecReturnData => _kSecReturnData.value;
-
-  set kSecReturnData(CFStringRef value) => _kSecReturnData.value = value;
-
-  late final ffi.Pointer<CFStringRef> _kSecValueData =
-      _lookup<CFStringRef>('kSecValueData');
-
-  CFStringRef get kSecValueData => _kSecValueData.value;
-
-  set kSecValueData(CFStringRef value) => _kSecValueData.value = value;
-
-  int SecItemCopyMatching(
-    CFDictionaryRef query,
-    ffi.Pointer<CFTypeRef> result,
-  ) {
-    return _SecItemCopyMatching(
-      query,
-      result,
-    );
-  }
-
-  late final _SecItemCopyMatchingPtr = _lookup<
-      ffi.NativeFunction<
-          OSStatus Function(
-              CFDictionaryRef, ffi.Pointer<CFTypeRef>)>>('SecItemCopyMatching');
-  late final _SecItemCopyMatching = _SecItemCopyMatchingPtr.asFunction<
-      int Function(CFDictionaryRef, ffi.Pointer<CFTypeRef>)>();
-
-  int SecItemAdd(
-    CFDictionaryRef attributes,
-    ffi.Pointer<CFTypeRef> result,
-  ) {
-    return _SecItemAdd(
-      attributes,
-      result,
-    );
-  }
-
-  late final _SecItemAddPtr = _lookup<
-      ffi.NativeFunction<
-          OSStatus Function(
-              CFDictionaryRef, ffi.Pointer<CFTypeRef>)>>('SecItemAdd');
-  late final _SecItemAdd = _SecItemAddPtr.asFunction<
-      int Function(CFDictionaryRef, ffi.Pointer<CFTypeRef>)>();
-
-  int SecItemUpdate(
-    CFDictionaryRef query,
-    CFDictionaryRef attributesToUpdate,
-  ) {
-    return _SecItemUpdate(
-      query,
-      attributesToUpdate,
-    );
-  }
-
-  late final _SecItemUpdatePtr = _lookup<
-      ffi.NativeFunction<
-          OSStatus Function(
-              CFDictionaryRef, CFDictionaryRef)>>('SecItemUpdate');
-  late final _SecItemUpdate = _SecItemUpdatePtr.asFunction<
-      int Function(CFDictionaryRef, CFDictionaryRef)>();
-
-  int SecItemDelete(
-    CFDictionaryRef query,
+  void CFRelease1(
```

Comment:
*Note:* There are some duplicate methods being generated (`CFRelease` and `CFRelease1`) with ffigen 5.0.0. They aren't an issue as they are unused. I opened an issue in the ffigen repo to figure out why though. Assuming it gets resolved in a later version, we can update and regenerate.